src/wmnds/components/icon/_icon.njk

```diff
@@ -1,6 +1,6 @@
 {% macro wmndsIcon(params) %}
 
-<svg {% if params.class %}class="{{params.class}}"{% endif %}>
+<svg {% if params.class %}class="{{params.class}}"{% endif %} {% if params.alt %}alt="{{params.alt}}"{% endif %} >
```

Comment:
SVGs don't have alts, instead they have titles.
See *Inline SVG*: http://web-accessibility.carnegiemuseums.org/code/svg/#:~:text=Inline%20SVG&text=The%20tag%20for%20an,to%20the%20tag.

Reply:
Oh god!😑 will update it quickly!